pkg/generators/markers_test.go

```diff
@@ -112,7 +112,7 @@ func TestParseCommentTags(t *testing.T) {
 				"+k8s:validation:maximum=20.0",
 				"+k8s:validation:minimum=30.0",
 			},
-			expectedError: `cannot unmarshal array into Go struct field CommentTags.minimum of type float64`,
+			expectedError: `failed to parse marker comments: cannot have multiple values for key 'minimum'`,
```

Comment:
Nice, outputting a more precise error message is great

Reply:
question: Is this still part of the PR?

Reply:
yes